examples/example-kotlin/android/helloworld/app/src/main/kotlin/io/grpc/helloworldexample/HelloworldActivity.kt

```diff
@@ -31,33 +33,38 @@ import io.grpc.ManagedChannel
 import io.grpc.ManagedChannelBuilder
 import io.grpc.examples.helloworld.GreeterGrpc
 import io.grpc.examples.helloworld.HelloRequest
+import kotlinx.android.synthetic.main.activity_helloworld.grpc_response_text
+import kotlinx.android.synthetic.main.activity_helloworld.send_button
+import kotlinx.android.synthetic.main.activity_helloworld.host_edit_text
+import kotlinx.android.synthetic.main.activity_helloworld.message_edit_text
+import kotlinx.android.synthetic.main.activity_helloworld.port_edit_text
 import java.io.PrintWriter
 import java.io.StringWriter
 import java.lang.ref.WeakReference
 import java.util.concurrent.TimeUnit
-import kotlinx.android.synthetic.main.activity_helloworld.*
```

Comment:
This is an excessive change, `kotlinx.android.synthetic.main.activity_helloworld.*` is better.